test/cases/measure/data/data.go

```diff
@@ -26,16 +26,16 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
-	g "github.com/onsi/ginkgo/v2"
-	gm "github.com/onsi/gomega"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
ci might fails

Reply:
> ci might fails

Oh, I see the linter issues... Let me revert back this part

Reply:
But It seems we miss format for the test package?

Reply:
No. It doesn't ignore it. But we have to make the order deterministic.